dart/lib/src/sentry_envelope_item.dart

```diff
@@ -11,11 +11,15 @@ class SentryEnvelopeItem {
   SentryEnvelopeItem(this.header, this.dataFactory);
 
   factory SentryEnvelopeItem.fromAttachment(SentryAttachment attachment) {
-    final cachedItem = _CachedItem(() async {
-      return await attachment.bytes;
-    });
+    final cachedItem = _CachedItem(() async => await attachment.bytes);
 
-    final getLength = () async => (await cachedItem.getData()).length;
+    final getLength = () async {
+      try {
+        return (await cachedItem.getData()).length;
+      } catch (_) {
+        return 0;
```

Comment:
an attachment that has problems getting its data should not be added to the envelope, so returning 0 looks like a valid attachment, and its wrong.

Reply:
maybe we return -1 here and who calls this function test if its -1 and drop it? we don't want to add an empty line or break the envelope